randovania/games/dread/logic_database/Burenia.txt

```diff
@@ -303,7 +303,12 @@ Extra - asset_id: collision_camera_002
                   Before Burenia - Dairon Hub Water Blob Right and Before Burenia - Dairon Hub Water Blob Left and Enabled Highly Dangerous Logic
                   Simple IBJ or Use Spin Boost
   > Event - Water Blob Right
-      Wave Beam or Lay Bomb or Lay Power Bomb
+      Any of the following:
+          Wave Beam or Lay Bomb or Lay Power Bomb
```

Comment:
Unless we make it so all blobs get destroyed from PBs, this should probably account for Shoot Beam with them.

Reply:
Good call.